### PR TITLE
Expand temp store

### PR DESCRIPTION
**Pull request by bubakazouba**

### CODE REVIEW

**Review by arthurshir — APPROVED** [review]
LGTM from tutorial FSM perspective

File: botfunctions/reminderactions.js [hunk 1]
```diff
@@ -15,7 +15,14 @@ CUSTOM_SNOOZE_SCENE.enter(ctx => {
...
     let userId = ctx.chat.id;
-    let reminderId = UserManager.getUserTemporaryStore(userId);
+    let currentTransactionInfo = UserManager.getCurrentTransactionInfo(userId);
+    let reminderId;
+    if (!!currentTransactionInfo) {
+        reminderId = currentTransactionInfo.reminderId;
+    }
+    else {
+        return ctx.reply("Something went wrong, please try again", Extra.markup(Markup.removeKeyboard(true))).catch(UserManager.catchBlocks);
+    }
```
Comment (arthurshir) [inline botfunctions/reminderactions.js:25]: Nit:
- Bc this logic is repeated, what do you think about adding a function to `UserManager` that either returns `reminderId` or `undefined`? Or whatever else that is reasonable that would cut down on this repeated logic.

File: userManager.js [hunk 2]
```diff
@@ -88,29 +88,41 @@ module.exports = class UserManager {
...
-            let temporaryStore = users[id].temporaryStore;
-            delete users[id].temporaryStore;
-            return temporaryStore;
+            users[id].incrementNumberOfInvalidTimezonesSet();
+        }
+    }
+
+    static getNumberOfRemindersSet(id) {
+        if (UserManager.userExists(id)) {
+            return users[id].getNumberOfRemindersSet();
```
Comment (arthurshir) [inline userManager.js:125]: Just thought of it, should we add tests for userManager?

**Review by arthurshir — APPROVED** [review]
Good with some nit comments.

File: userManager.js (hunk 2 above)
Comment (bubakazouba) [reply userManager.js:125]: added tests for user.js instead

**Pull request merged**